packages/gatsby-theme/src/components/Share/ShareTwitter.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types"
+import { TwitterShareButton } from "react-share"
+import Icon from "../Icon"
+
+const ShareTwitter = ({ title, to, handler, hashtags, children, ...props }) => {
+  return (
+    <TwitterShareButton
+      url={to}
+      title={title}
+      via={handler}
+      hashtags={hashtags}
+      resetButtonStyle={false}
+      {...props}
+    >
+      <Icon icon='twitter' />
+      {children}
+    </TwitterShareButton>
+  )
```

Comment:
Same refactoring of using a single component with a Hashmap to resolve the specific component from the "react-share" library